src/main/drivers/serial_uart_stm32f4xx.c

```diff
@@ -52,11 +52,9 @@ typedef struct uartDevice_s {
     ioTag_t tx;
     volatile uint8_t rxBuffer[UART_RX_BUFFER_SIZE];
     volatile uint8_t txBuffer[UART_TX_BUFFER_SIZE];
-    uint32_t rcc_ahb1;
     rccPeriphTag_t rcc_apb2;
```

Comment:
rccPeriphTag_t encodes bus type, so `rcc_uart` can replace  rcc_apb2 and rcc_apb1